### PR TITLE
UHM-4026 Hide Mental Health dashboard widgets for Haiti

### PR DESCRIPTION
**Pull request by brandones**
https://pihemr.atlassian.net/browse/UHM-4026

### CODE REVIEW

File: api/src/main/java/org/openmrs/module/mirebalais/apploader/CustomAppLoaderFactory.java [hunk 1]
```diff
@@ -2101,34 +2101,36 @@ private void enableHypertensionProgram() {
...
-                        "maxResults", "12"
-                )),
-                "coreapps", "dashboardwidgets/dashboardWidget"));
-
-        if (!config.getCountry().equals(ConfigDescriptor.Country.LIBERIA)) {
-            apps.add(addToMentalHealthDashboardSecondColumn(app(Apps.GAD7_GRAPH,
-                    "pih.app.gad7.graph.title",
+        if (!config.getCountry().equals(ConfigDescriptor.Country.HAITI)) {
+            apps.add(addToMentalHealthDashboardSecondColumn(app(Apps.PHQ9_GRAPH,
+                    "pih.app.phq9.graph.title",
```
Comment (brandones) [inline api/src/main/java/org/openmrs/module/mirebalais/apploader/CustomAppLoaderFactory.java]: this is a very bad diff

Comment (brandones) [reply api/src/main/java/org/openmrs/module/mirebalais/apploader/CustomAppLoaderFactory.java]: try "ignore whitespace changes"

**Comment by brandones** [comment]
Fair. Though I'd much rather get rid of them by changing the thing completely, abolish all this component-site-branching.

**Comment by mogoodrich** [comment]
> Fair. Though I'd much rather get rid of them by changing the thing completely, abolish all this component-site-branching.

yeah @brandones I was going to add that... but, until then... :)

**Pull request merged**